### PR TITLE
Middleware 1.3.23

### PR DESCRIPTION
**Pull request by TSchmiedlechner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ These release notes will contain a sum-up of all customer-relevant changes that
...
 If you're viewing this on GitHub, please navigate to https://docs.fiskaltrust.cloud for a more convenient view of this content and open the section of interest to the left. If you have any questions regarding the changes described here, please directly reach out to us via [info@fiskaltrust.de](mailto:info@fiskaltrust.de) / [info@fiskaltrust.at](mailto:info@fiskaltrust.at) / [info@fiskaltrust.fr](mailto:info@fiskaltrust.fr).
 
+## September 15, 2021
+
+### Middleware 1.3.23 (Germany)
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+[Middleware 1.3.23 Release Notes](middleware/middleware-1.3.23.md)
+
+
```
Comment (StefanKert) [inline README.md:20]: do we always have two linebreaks?

File: middleware/middleware-1.3.23.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Flag optional signature items
+Starting with this version, we're adding the _ftSignatureFormatFlag_ `0x0000000000010000` to signatures that are not mandatory to print. Especially with the recently changed _KassenSichV_ regulations in Germany (which allow to omit printing most texts when a QR code is added), this should enable PosCreators to more easily decide which information needs to be printed onto the receipt and which information is optional.
+
+More information can be found in the [respective docs](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftsignatureformat). Please note that this behavior can be deactivated by setting the Queue configuration parameter `FlagOptionalSignatures` to `false`.
```
Comment (StefanKert) [inline middleware/middleware-1.3.23.md:21]: I would probably highlight the last section in an extra box e.g. "If this causes any problems you can.."

File: middleware/middleware-1.3.23.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+It's available in the [German Portal's download section](https://portal.fiskaltrust.de/AccountProfile/Download).
+
+## Affected packages
+Packages not listed here were not updated, as we decided to not increase the version of unchanged packages. All packages with versions greater or equal to 1.3.1 are compatible with each other (it is e.g. possible to use _fiskaltrust.Middleware.SCU.Swissbit.1.3.1_ with the new queue packages).
+
+- _fiskaltrust.Middleware.Queue.EF v1.3.23_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.23_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.23_
+- _fiskaltrust.Middleware.SCU.DE.FiskalyCertified v1.3.23_
```
Comment (StefanKert) [inline middleware/middleware-1.3.23.md:63]: Do we want to highlight that this is the first release that is not a release candidate?

File: README.md (hunk 1 above)
Comment (TSchmiedlechner) [reply README.md:20]: Yep, only before new days though - it's easier to read through the file IMO 😄

File: middleware/middleware-1.3.23.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+title: Version 1.3.23
+---
+
+# fiskaltrust.Middleware 1.3.23 (Germany)
+_September 15, 2021_
+
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+
+:::caution
```
Comment (fiscoal) [inline middleware/middleware-1.3.23.md:12]: Is this being replaced somewhere?

File: middleware/middleware-1.3.23.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Starting with this version, we're adding the _ftSignatureFormatFlag_ `0x0000000000010000` to signatures that are not mandatory to print. Especially with the recently changed _KassenSichV_ regulations in Germany (which allow to omit printing most texts when a QR code is added), this should enable PosCreators to more easily decide which information needs to be printed onto the receipt and which information is optional.
+
+More information can be found in the [respective docs](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftsignatureformat). Please note that this behavior can be deactivated by setting the Queue configuration parameter `FlagOptionalSignatures` to `false`.
+
+## Feature: Automatically update FCC with SCU update
+Starting from this version, each _SwissbitCloud_ and _DeutscheFiskal_ SCU are connected to a specific minimal Fiskal Cloud Connector (FCC) version. If the currently installed FCC version is lower than this, the Middleware will automatically download and update the newer FCC. This should enable users that have already installed previous versions to profit from the latest improvements in this external dependency.
```
Comment (fiscoal) [inline middleware/middleware-1.3.23.md:30]:
```suggestion
Starting from this version, each _SwissbitCloud_ and _DeutscheFiskal_ SCU is tied to a specific minimal Fiskal Cloud Connector (FCC) version. If the currently installed FCC version is lower than this, the Middleware will automatically download and update the newer FCC. This should enable users that have already installed previous versions to profit from the latest improvements in this external dependency.
```

File: middleware/middleware-1.3.23.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+_September 15, 2021_
+
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
```
Comment (fiscoal) [inline middleware/middleware-1.3.23.md:16]: Anything missing?

File: middleware/middleware-1.3.23.md (hunk 3 above)
Comment (fiscoal) [inline middleware/middleware-1.3.23.md:69]: Is fiskaly 1 completely out of the picture at this point?

File: middleware/middleware-1.3.23.md (hunk 2 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md]: Valid, thanks! Added a "hint" box.

File: middleware/middleware-1.3.23.md (hunk 3 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md:69]: Good point, I've added a sentence to the intro 🙂

**Review by StefanKert — APPROVED** [review]

File: middleware/middleware-1.3.23.md (hunk 4 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md:12]: Yep, this is handled by Docusaurus (https://docusaurus.io/docs/markdown-features/admonitions)

File: middleware/middleware-1.3.23.md (hunk 3 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md:69]: No updates included in v1 this time, yes 👍

File: middleware/middleware-1.3.23.md (hunk 6 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md:16]: I don't think so, this ::: are automatically converted by Docusaurus (see above)

File: middleware/middleware-1.3.23.md (hunk 5 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.23.md:30]: PR was already merged, but I fixed it in `master` - thanks!

**Pull request merged**